### PR TITLE
Make props bindable

### PR DESCRIPTION
**Pull request by DNA-PC**
Make all props bindable instead of static.

Misc changes:
- Add .nvmrc (best practice)
- Ignore .idea folder (.gitignore)
- Npm packages updates (due to build, could be ignored/removed from PR, I guess)

### CODE REVIEW

File: ww-config.js [hunk 1]
```diff
@@ -162,6 +174,7 @@ export default {
                 ],
             },
             defaultValue: 'vuecal--gray-theme',
+            bindable: true,
```
[the remark this answers is not in the document]
Comment (DNA-PC) [reply ww-config.js]: Oversight on my side, fixed.

File: ww-config.js [hunk 2]
```diff
@@ -366,6 +403,7 @@ export default {
                 en: 'Id property',
             },
             type: 'ObjectPropertyPath',
```
[the remark this answers is not in the document]
Comment (DNA-PC) [reply ww-config.js]: Fixed

**Review by DNA-PC — COMMENTED** [review]
All requested changes done

**Review by AlexisJC — APPROVED** [review]
@AurelieV @kevinbarfleur Good for me

**Pull request closed without merging**